ejercicio_2/spec/gist_content_spec.rb

```diff
@@ -0,0 +1,62 @@
+require_relative '../gist_content'
+require 'spec_helper'
+
+path = '../ejercicio_2/spec/test'
```

Comment:
Try using `let` to define variables instead. You can read more about it here https://dzone.com/articles/rspec-let-vs-before or here https://www.betterspecs.org/#let